device_doctor/lib/src/process_helper.dart

```diff
@@ -67,3 +69,50 @@ Future<String> eval(String executable, List<String> arguments,
 
   return output.trimRight();
 }
+
+/// Splits [from] into lines and selects those that contain [pattern].
+Iterable<String> grep(Pattern pattern, {@required String from}) {
+  return from.split('\n').where((String line) {
+    return line.contains(pattern);
+  });
+}
+
+Future<void> killAllRunningProcessesOnWindows(String processName) async {
+  ProcessManager processManager = LocalProcessManager();
+  while (true) {
+    final pids = runningProcessesOnWindows(processName);
+    if (pids.isEmpty) {
+      return;
+    }
+    for (String pid in pids) {
+      processManager.runSync(<String>['taskkill', '/pid', pid, '/f']);
+    }
+    // Killed processes don't release resources instantenously.
+    await Future<void>.delayed(Duration(seconds: 1));
```

Comment:
nit: Move to variable to prevent this being a magic number.

Reply:
Done.